index.html

```diff
@@ -1,54 +1,378 @@
+
+            <div class="language-list-items">
+              <ul>
+                <li><a href="#">Javascript</a></li>
+                <li><a href="#">Ruby</a></li>
+                <li><a href="#">Html</a></li>
+                <li><a href="#">Css</a></li>
+              </ul>
+            </div>
+          </div>
+
+          <!-- Frameworks heading start from here -->
+
+          <div class="framework-heading">
+            <ul>
+              <li><a href="#">Frameworks</a></li>
+            </ul>
+
+            <div class="framework-list-items">
+              <ul>
+                <li><a href="#">Bootstrap</a></li>
+                <li><a href="#">Ruby on Rails</a></li>
+                <li><a href="#">RSPec</a></li>
+                <li><a href="#">CapyBara</a></li>
+                <li><a href="#">Selenium</a></li>
+              </ul>
+            </div>
+          </div>
+
+          <!-- Skills Heading start from here -->
+
+          <div class="skills-heading">
+            <ul>
+              <li><a href="#">Skills</a></li>
+            </ul>
+
+            <div class="skill-list-items">
+              <ul>
+                <li><a href="#">Codekit</a></li>
+                <li><a href="#">Github</a></li>
+                <li><a href="#">Codepen</a></li>
+                <li><a href="#">Gitlab</a></li>
+                <li><a href="#">Terminal</a></li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </section>
+
+        <!-- Contact-section start from here -->
+
+      <section class="contact-section">
+        <div class="contact-heading">
+          <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+        </div>
+        <div class="form-container">
+          <form action="#" class="form">
+            <input type="text" name="name" id="name" placeholder="Full name">
+            <input type="email" name="email" id="email" placeholder="Email address">
+            <textarea name="textarea" id="textarea" cols="5" rows="5" placeholder="Enter text here"></textarea>
+            <button class="form-button">Get in touch</button>
+          </form>
+        </div>
+      </section>
+
+        <!-- Footer-section start from here -->
+
+      <footer class="footer-container">
+        <div class="footer-icon-list">
           <ul>
-            <li><a href="#"><i class="fa fa-github social-icons" aria-hidden="true"></i></a></li>
-            <li><a href="#"><i class="fa fa-linkedin social-icons" aria-hidden="true"></i></a></li>
-            <li><a href="#"><i class="fa fa-angellist social-icons" aria-hidden="true"></i></a></li>
-            <li><a href="#"><i class="fa fa-twitter social-icons" aria-hidden="true"></i></a></li>
-            <li><a href="#" class="vectorimg"><img src="/images/Vector.png" class="vectorimg" alt="Vector"></a></li>
+            <li><a href="#"><i class="fa fa-github footer-icon" aria-hidden="true"></i></a></li>
```

Comment:
This is your design:
![image](https://user-images.githubusercontent.com/61361037/212378879-34566aa1-5c19-4f34-97c0-7b5d5ac75656.png)
This is Figma design
![image](https://user-images.githubusercontent.com/61361037/212378971-e03c8273-b377-4e9a-8f5f-71680e2c6d68.png)


Kindly add this line.